### ISSUE TITLE
"lexicographical \> is undefined" is missing a wiki page

### ISSUE THREAD

**Issue by jyn514** [koalaman/shellcheck#2192]
#### For bugs
- Rule Id (if any, e.g. SC1000): SC3012
- My shellcheck version (`shellcheck --version` or "online"): online
- [x] The rule's wiki page does not already cover this (e.g. https://shellcheck.net/wiki/SC2086)
- [x] I tried on https://www.shellcheck.net/ and verified that this is still a problem on the latest commit

#### Here's a snippet or screenshot that shows the problem:

```sh
#!/bin/sh
x=1.31
y=1.29
! [ $x \> $y ]
```

#### Here's what shellcheck currently says:

```
Line 4:
! [ $x \> $y ]
       ^-- SC3012: In POSIX sh, lexicographical \> is undefined.
```

#### Here's what I wanted or expected to see:

A wiki page at https://github.com/koalaman/shellcheck/wiki/SC3012 explaining why `\>` is undefined (it works fine in `dash`) and what I should replace it with.

**Comment by TinCanTech** [comment]
What `shellcheck` means is that `>` (and `\>`) are undefined in POSIX,
See: https://pubs.opengroup.org/onlinepubs/9699919799/utilities/test.html#tag_20_128

Sure, `dash` lets you use this but it's not POSIX.

If it bothers you then you can add a `shellcheck` directive to ignore it.

You can also create the wiki page yourself.

**Comment by jyn514** [comment]
What can I replace it with? I think `expr` will work, but there's also a lint against that: https://github.com/koalaman/shellcheck/wiki/SC2003

**Comment by brother** [comment]
How to solve it depends on what your shell can do and what values you are comparing. Not sure there are any complete solutions. As @TinCanTech mentioned the wiki is open for edits or additions. 

I've added a skeleton page base on the one for SC3014.

**Comment by koalaman** [comment]
`expr` will indeed work, and I've updated the wiki page. ShellCheck does not warn about using `expr` for features that are otherwise hard to replicate in POSIX sh, such as `<` and its variation, plus `:`. 

Thanks all!
